In-house patch to remove unnecessary dependencies from Cinder's
requirements files. The specific reasons are as follows:

oslo.rootwrap	Not applicable to Solaris

pycrypto	Not applicable to Solaris (various drivers specific)

rtslib-fb	Not applicable to Solaris (Linux iSCSI specific)

--- cinder-8.0.0.0rc2/requirements.txt.~1~	2016-03-29 06:02:44.000000000 -0700
+++ cinder-8.0.0.0rc2/requirements.txt	2016-03-30 02:08:26.568745142 -0700
@@ -22,7 +22,6 @@ oslo.messaging>=4.0.0 # Apache-2.0
 oslo.middleware>=3.0.0 # Apache-2.0
 oslo.policy>=0.5.0 # Apache-2.0
 oslo.reports>=0.6.0 # Apache-2.0
-oslo.rootwrap>=2.0.0 # Apache-2.0
 oslo.serialization>=1.10.0 # Apache-2.0
 oslo.service>=1.0.0 # Apache-2.0
 oslo.utils>=3.5.0 # Apache-2.0
@@ -31,7 +30,6 @@ osprofiler>=1.1.0 # Apache-2.0
 paramiko>=1.16.0 # LGPL
 Paste # MIT
 PasteDeploy>=1.5.0 # MIT
-pycrypto>=2.6 # Public Domain
 pyparsing>=2.0.1 # MIT
 python-barbicanclient>=3.3.0 # Apache-2.0
 python-glanceclient>=2.0.0 # Apache-2.0
@@ -44,7 +42,6 @@ retrying!=1.3.0,>=1.2.3 # Apache-2.0
 Routes!=2.0,!=2.1,>=1.12.3;python_version=='2.7' # MIT
 Routes!=2.0,>=1.12.3;python_version!='2.7' # MIT
 taskflow>=1.26.0 # Apache-2.0
-rtslib-fb>=2.1.41 # Apache-2.0
 simplejson>=2.2.0 # MIT
 six>=1.9.0 # MIT
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
